libs/wire-api/src/Wire/API/Team/Feature.hs

```diff
@@ -35,10 +35,13 @@ module Wire.API.Team.Feature
     defaultClassifiedDomains,
 
     -- * Swagger
+
+    -- This is used in:
+    -- - this packge: in Wire.API.Swagger in 'models'
+    -- - Stern: in Stern.Swagger in 'sternModels', in Stern.API in a few places
```

Comment:
@fisx , does this answer your question on why we still need Swagger 1.2 in this module?